<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #E6E9ED; /* Gris claro, limpio y profesional */
}

QLabel {
    color: #2C3E50;
    font-size: 13px;
    font-family: 'Arial';
}

QLineEdit {
    background-color: #FFFFFF;
    color: #000000;
    border: 1px solid #BDC3C7;
    border-radius: 4px;
    padding: 5px;
    font-size: 12px;
}

QPushButton {
    background-color: #0078D7; /* Azul estilo Windows/Asterisk */
    color: white;
    border: none;
    border-radius: 4px;
    padding: 6px 12px;
    font-size: 13px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #005A9E;
}

QPushButton:pressed {
    background-color: #003F6B;
}

QGroupBox {
    border: 1px solid #CCCCCC;
    border-radius: 5px;
    margin-top: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px;
    color: #2C3E50;
    font-weight: bold;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">QMainWindow {
    background-color: #E6E9ED; /* Fondo gris claro */
}

/* Etiquetas */
QLabel {
    color: #2C3E50;
    font-size: 13px;
    font-family: 'Arial';
}

/* Campos de texto */
QLineEdit {
    background-color: #FFFFFF;
    color: #000000;
    border: 1px solid #BDC3C7;
    border-radius: 4px;
    padding: 5px;
    font-size: 12px;
}

/* Botón de iniciar sesión (azul) */
#login_btn {
    background-color: #0078D7;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 6px 12px;
    font-size: 13px;
    font-weight: bold;
}
#login_btn:hover {
    background-color: #005A9E;
}
#login_btn:pressed {
    background-color: #003F6B;
}

/* Botón catálogo de productos (verde) */
#catalog_product_btn {
    background-color: #5CB85C;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 6px 12px;
    font-size: 13px;
    font-weight: bold;
}
#catalog_product_btn:hover {
    background-color: #4CAE4C;
}
#catalog_product_btn:pressed {
    background-color: #449D44;
}

/* Botón catálogo de clientes (naranja) */
#catalog_client_btn {
    background-color: #F0AD4E;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 6px 12px;
    font-size: 13px;
    font-weight: bold;
}
#catalog_client_btn:hover {
    background-color: #EC971F;
}
#catalog_client_btn:pressed {
    background-color: #D58512;
}
</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>-8</y>
          <width>280</width>
          <height>284</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0" colspan="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>278</width>
             <height>58</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>58</width>
             <height>168</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLineEdit" name="user_edit">
               <property name="placeholderText">
                <string>Usuario</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="pass_edit">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Contraseña</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="login_btn">
             <property name="text">
              <string>Iniciar sesión </string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="cancel_btn">
             <property name="text">
              <string>Cancelar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>48</width>
             <height>168</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0" colspan="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>278</width>
             <height>48</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="catalog_product_btn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Catálogo de productos</string>
        </property>
       </widget>
       <widget class="QPushButton" name="catalog_client_btn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>120</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Catálogo de clientes</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>296</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
